internal/services/network/resourceids.go

```diff
@@ -38,3 +38,4 @@ package network
 
 // Network
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=NetworkInterfaceIpConfiguration -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Network/networkInterfaces/networkInterface1/ipConfigurations/config1
+//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=VirtualNetworkPeering -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Network/virtualNetworks/network1/virtualNetworkPeerings/virtualNetworkPeering1
```

Comment:
This is a legacy pattern for generating resource ID parsers/validators in the provider. The parsers and validators that we use are contained within the SDK now, please see our [contributor guide on Resource IDs](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-resource-ids.md#resource-id-parsers-and-validators-in-hashicorpgo-azure-sdk) for more information
```suggestion
```